probe/tag/docker_tagger.go

```diff
@@ -157,15 +179,15 @@ func (t *DockerTagger) update() bool {
 	}
 }
 
-func (t *DockerTagger) updateContainers(client dockerClient) error {
-	apiContainers, err := client.ListContainers(docker.ListContainersOptions{All: true})
+func (t *DockerTagger) updateContainers() error {
+	apiContainers, err := t.client.ListContainers(docker.ListContainersOptions{All: true})
 	if err != nil {
 		return err
 	}
 
 	containers := []*docker.Container{}
```

Comment:
t.update never returns unless there's an error during set up, or the user issues a quit. But you call it repeatedly in loop, and always on a ticker. Why?